packages/now-cli/src/commands/deploy/index.js

```diff
@@ -112,89 +84,5 @@ export default async ctx => {
     }
   }
 
-  const file = highlight('now.json');
-  const prop = code('version');
-
-  if (localConfig) {
-    const { version } = localConfig;
-
-    if (version) {
-      if (typeof version === 'number') {
-        if (version !== 1 && version !== 2) {
-          const first = code(1);
-          const second = code(2);
-
-          output.error(
```

Comment:
I think we might want to keep an error here when `typeof version === 'number' && version !== 2` so that we can fail if someone tries `version: 3` for example.

Reply:
And maybe we can print a helpful error if they have ` version: 1` that says its no longer supported and links to the upgrade docs.

Reply:
The API would return an error in that case. Ideally we should remove all the validation from Now CLI and let the API handle everything => less duplicated code, no new version when updating error messages etc.